From d1059124f98bde9d0b12cfc21a84a81b940a7281 Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Tue, 11 Jan 2011 18:01:03 +0800
Subject: ENGR00137979-3 add header file for performance monitor driver

	add header file for performance monitor driver under include/linux

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 include/linux/fsl_devices.h |   11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index fa8c145..7e41d6d 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -293,4 +293,15 @@ int fsl_deep_sleep(void);
 static inline int fsl_deep_sleep(void) { return 0; }
 #endif
 
+struct mxs_perfmon_bit_config {
+	int reg;
+	int field;
+	const char *name;
+};
+
+struct mxs_platform_perfmon_data {
+	struct mxs_perfmon_bit_config *bit_config_tab;
+	int bit_config_cnt;
+};
+
 #endif /* _FSL_DEVICE_H_ */
-- 
1.7.9.5

